src/olympia/templates/photon-footer.html

```diff
@@ -0,0 +1,73 @@
+<footer class="Footer">
```

Comment:
This was based on the html from frontend - with a few changes for urls and includes for the template.